autoemulate/experimental/learners/base.py

```diff
@@ -486,9 +213,16 @@ def fit(self, *args):
         self.metrics["r2"].append(r2_val)
         self.metrics["rate"].append(self.n_queries / (len(self.metrics["rate"]) + 1))
         self.metrics["n_queries"].append(self.n_queries)
-        self.metrics["trace"].append(self.trace(Sigma, self.out_dim).item())
-        self.metrics["logdet"].append(self.logdet(Sigma, self.out_dim).item())
-        self.metrics["max_eigval"].append(self.max_eigval(Sigma).item())
+
+        # If Gaussian output
+        if isinstance(output, MultivariateNormal):
+            assert isinstance(output.variance, TensorLike)
+            assert output.variance.ndim == 2
+            assert output.variance.shape[1] == self.out_dim
+            std = torch.sqrt(output.variance)
```

Comment:
Revised to use get the standard deviation here aiming to be consistent with previous GP impl used here.

Reply:
Revised to use `.variance` property of `MultivariateNormal` (which is the expected diagonal of the covariance matrix). Check this as part of #430 and has implications for #434.